public/docs/_examples/animations/ts/src/hero-list-classes.component.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Component,
+  Input,
+  animation,
+  state,
+  style,
+  animate,
+  transition,
+  group
+} from '@angular/core';
+import {Hero, Heroes} from './hero.service';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hero-list-classes',
+  template: `
+    <div class="hero"
+         *ngFor="let hero of heroes.get()"
+         @heroState="hero.state"
+         (click)="hero.toggleActivation()">
+      {{hero.name}}
+    </div>
+  `,
+  styleUrls: ['hero-list.component.css'],
+  // #docregion animationdef
+  animations: [
+    animation('heroState', [
+      state('inactive', style('.inactive')),
```

Comment:
These are not ready to use yet since the CSS parser has not been integrated into animations just yet.